app/Views/ScBooking/AvailableTimes.cshtml

```diff
@@ -13,102 +11,59 @@
     </div>
     <hr />
     <h6>Step 3</h6>
-    <h2>Book Your Hearing</h2>
+    <h2>Choose Trial Date(s)</h2>
```

Comment:
Just noticed on a call with Winnie, this needs to be dynamic or it will say "Choose Trial Date(s)" even for conference bookings. I'll update that shortly

Reply:
(updated!)